### PR TITLE
Changed the wording of context attributes to make it clearer

### PR DESCRIPTION
**Pull request by schatekar**
Changed the wording to make it clearer what is being designed and what is being considered as a possible use of context attributes

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -118,11 +118,12 @@ Every event in the OpenEventing specification includes context which is a set
...
 
-Context is designed to be delivered separately from the event data (e.g. in
-protocol headers or protocol specific attributes).  This allows the context
-to be analyzed without having to deserialize the event data.  The context can
-also be serialized with the event data (e.g. a JSON implementation may
-consist of one JSON object that contains both context and data).
+Context is designed such that it can be delivered separately from the event data 
+(e.g. in protocol headers or protocol specific attributes).  This allows the context
+to be inspected at the destination without having to deserialize the event data.  
+The context may also need to be serialized with the event data for some use cases
```
Comment (duglin) [inline spec.md]: I know we haven't converted over to RFC2119 keywords yet, but we might as well start... can you change this "may" to a "MAY" ?  And the "may" on the next line to "might".

File: spec.md [hunk 2]
```diff
@@ -118,11 +118,12 @@ Every event in the OpenEventing specification includes context which is a set
...
+(e.g. a JSON implementation may consist of one JSON object that contains both context
```
Comment (duglin) [inline spec.md]: I think "use" might be better than "consist of".

**Comment by duglin** [comment]
thanks
LGTM

**Comment by ac360** [comment]
LGTM.  Thanks @schatekar

**Comment by duglin** [comment]
@schatekar can I get you to sign your commit since we're using DCOs?

**Comment by duglin** [comment]
reopening to kick travis

**Comment by duglin** [comment]
Approved on today's call by unanimous consent

**Pull request merged**